jinjang/templates/page.html

```diff
@@ -14,20 +14,20 @@
 	<div class="wp-block-column">
 		<!-- wp:group {"style":{"spacing":{"padding":{"top":"var:preset|spacing|70","bottom":"var:preset|spacing|70"}},"dimensions":{"minHeight":"100vh"}},"layout":{"type":"flex","orientation":"vertical","justifyContent":"stretch","verticalAlignment":"space-between"}} -->
 		<div class="wp-block-group" style="min-height:100vh;padding-top:var(--wp--preset--spacing--70);padding-bottom:var(--wp--preset--spacing--70)">
-			<!-- wp:group {"layout":{"type":"constrained"}} -->
-			<div class="wp-block-group">
-				<!-- wp:group {"tagName":"main","layout":{"type":"constrained","justifyContent":"center"}} -->
-				<main class="wp-block-group">
+			<!-- wp:group {"tagName":"main","layout":{"type":"constrained"}} -->
+			<main class="wp-block-group">
+				<!-- wp:group {"layout":{"type":"constrained","justifyContent":"center"}} -->
+				<div class="wp-block-group">
 					<!-- wp:post-featured-image /-->
 
 					<!-- wp:post-title /-->
 
 					<!-- wp:post-content {"lock":{"move":false,"remove":false},"layout":{"type":"constrained","justifyContent":"center"}} /-->
-				</main>
+				</div>
 				<!-- /wp:group -->
 
-				<!-- wp:pattern {"slug":"jinjang/comments"} /-->
-			</div>
+				<!-- wp:template-part {"slug":"comments"} /-->
```

Comment:
The comments template part needs to be included.

<img width="1108" alt="image" src="https://github.com/Automattic/themes/assets/1157901/2b7f8421-da1d-4701-985e-abd95d2ef3d1">

Reply:
Sorry, what am I missing? I can't replicate the missing comment template on `page.html` and `single.html`. I tested it with/without Gutenberg, but it worked for me.

![Screenshot 2023-10-13 at 09 32 06](https://github.com/Automattic/themes/assets/908665/d8e93f4d-4471-4b70-afb1-0590e9f64d16)